tools/tm-bench/transacter.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"crypto/md5"
+	"crypto/sha256"
 	"encoding/binary"
 	"encoding/hex"
 	"encoding/json"
```

Comment:
forgot to add

```
// it is ok here to use math/rand; we do not need cryptographically secure random
// number generator here and test run a bit faster
```

here too

Reply:
Thanks! Added and slightly changed to 
```
// it is ok to use math/rand here: we do not need a cryptographically secure random
// number generator here and we can run the tests a bit faster
```
in all places